packages/orchestration/src/proposals/orchestration-proposal.js

```diff
@@ -92,6 +92,13 @@ export const getManifestForOrchestration = (_powers, { orchestrationRef }) => ({
         orchestrationVat: 'orchestration',
       },
     },
+
+    [addOrchestrationToClient.name]: {
```

Comment:
I gather this is for use from the REPL.

Do we test it?

I'm surprised the `homeKeys` test hasn't failed.
https://github.com/Agoric/agoric-sdk/blob/master/packages/boot/test/bootstrapTests/test-demo-config.ts#L42

Another candidate is..

https://github.com/Agoric/agoric-sdk/blob/master/packages/solo/test/test-home.js

Reply:
Thank you for the pointers, but I have removed this from the PR